docs/Behaviour - Server Side.md

```diff
@@ -6,39 +6,29 @@ The initial state of Active Constraints of a Sender MUST be an empty `constraint
 
 Non-empty `constraint_sets` array MUST be treated as unordered unless at least one Constraint Set has `urn:x-nmos:cap:meta:preference` attribute.
 
-Once a Sender accepts proposed Active Constraints, this Sender, the Flow and the Connection API `/transportfile` resource (if used) associated with this Sender and the Source associated with this Flow MUST satisfy the Active Constraints when the Sender is active.
-
-### Request Methods
-
-- `GET` request returns the last successfully applied Active Constraints.
-- `PUT` request MUST be validated and if the Sender supports Parameter Constraints used in the proposed Active Constraints and is capable to adhere at least one of the Constraint Sets, then the associated Inputs may be reconfigured, which may eventually cause the update of the corresponding Flows, Sources, `/transportfile` and Sender. If successful, the operation MUST return the accepted Active Constraints, otherwise an error MUST be returned and nothing changed.
-- `DELETE` request MUST clear Active Constraints.
+Active Constraints are satisfied when any of its Constraint Sets are satisfied in terms of [BCP-004-01][BCP-004-01].
```

Comment:
I would keep this to make sure a Sender cannot advertise through  IS-04 a Flow or through IS-05 a transport file that violate the active constraints. If we simply state that a Sender cannot transmit to the network it keeps the possibility for a sender to advertise an illegal Flow and SDP transport file ...

"Once a Sender accepts proposed Active Constraints, this Sender, the Flow and the Connection API `/transportfile` resource (if used) associated with this Sender and the Source associated with this Flow MUST satisfy the Active Constraints when the Sender is active."

Reply:
I don't quite understand the point. A Sender has some Flow and transport file. Each time they change or Active Constraints change, the Node validates the Flow and the SDP against the Active Constraints. If validation fails, the Node prohibits transmitting with an illegal Flow and SDP. I don't understand, how we can avoid advertising an illegal Flow and SDP at all.

The removed paragraph doesn't save us from it as well because of "when the Sender is active" so I think "Preventing restrictions violation" in its current state covers this removed paragraph.

Reply:
I would not want a Sender to be allowed to publish an associated Flow/Source that violate the active constraints. Just not transmitting to the network it not enough. This paragraph provide more control over such behavior. This is why I think we must keep it. What would be the reason to remove it?

Reply:
I removed it to have this language under "Preventing restrictions violation" and it seems to me to be covered there. 

> Once a Sender accepts proposed Active Constraints, <...> Flow <...> MUST satisfy the Active Constraints **when the Sender is active**.

For me it means that either the published Flow is illegal and Sender can't transmit it or it's legal and Sender can transmit it. This logic is described in "Preventing restrictions violation" now. Could you please explain how this paragraph prevents from publishing an illegal Flow?

Reply:
Without this text "Once a Sender accepts proposed Active Constraints, this Sender, the Flow and the Connection API `/transportfile` resource (if used) associated with this Sender and the Source associated with this Flow MUST satisfy the Active Constraints when the Sender is active." the only restriction imposed by IS-11 is to not transmit a stream violating the active constraints over the network. IS-11 says nothing else. This is why I think it is important to include this text to add the additional restrictions that an active sender cannot publish an illegal flow ... only an inactive sender would be allowed.

Reply:
`Sender MUST NOT transmit` -> `Sender MUST become inactive` solved this issue, so I'm marking this as resolved.